src/browserlib/get-title.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gets the title of the document
+ */
+export default function () {
+  const title = window.document.querySelector('title');
+  if (window.location.href === 'https://html.spec.whatwg.org/multipage/workers.html') {
```

Comment:
in the brave new MoU world, do we still need that exception? (probably no need to fix it here and now)